File: in_Java/1317 Convert Integer to the Sum of Two No-Zero Integers.java
class Solution {
    public int[] getNoZeroIntegers(int n) {
        int[] res = new int[2];
        for(int i = 1; i < n; i++){
            if(check(i) && check(n - i)){
                res[0] = i;
                res[1] = n-i;
                return res;
            }
        }
        return res;
    }
    private boolean check(int n){
        for(char c : String.valueOf(n).toCharArray()){
            if(c == '0') return false;
        }
        return true;
    }
}

File: in_Python/1317 Convert Integer to the Sum of Two No-Zero Integers.py
class Solution(object):
    def check(self, n):
        while n != 0:
            if n % 10 == 0:
                return True
            n /= 10
        return False
    def getNoZeroIntegers(self, n):
        """
        :type n: int
        :rtype: List[int]
        """
        for i in range(1, n):           
            if self.check(i):
                continue
            else:
                if not self.check(n-i):
                    return [i, n-i]
        return None
        